mautic/mautic
# Devices and Push devices tab to contacts detail

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Just added another tabs to contacts detail.
Missing info about contact devices and also about push ids related to this PR https://github.com/mautic/mautic/pull/5286

#### Steps to test this PR:
1. Go to lead detail and see new tabs 

Devices
![image](https://user-images.githubusercontent.com/462477/32404656-5120aac6-c154-11e7-8e5b-c11c39b7c24a.png)

Push devices
![image](https://user-images.githubusercontent.com/462477/32404653-277c464e-c154-11e7-8540-51f55896c357.png)
